daprdocs/content/en/operations/security/mtls.md

```diff
@@ -163,9 +164,52 @@ helm install \
 If the Root or Issuer certs are about to expire, you can update them and restart the required system services.
 
 {{% alert title="Avoiding downtime when rotating certificates" color="warning" %}}
-To avoid downtime when rotating expiring certificates your new certificates must be signed with the same private root key as the previous certificates.
+To avoid downtime when rotating expiring certificates your new certificates must be signed with the same private root key as the previous certificates. This is not currently possible using self-signed certificates generated by Dapr.
+{{% /alert %}}
+
+#### Dapr-generated self-signed certificates
+
+1. Clear the existing Dapr Trust Bundle secret by saving the following YAML to a file (e.g. `clear-trust-bundle.yaml`) and applying this configuration.
```

Comment:
Suggest changing to `applying this secret` since we do have the concept of a Configuration also.

Reply:
```suggestion
1. Clear the existing Dapr Trust Bundle secret by saving the following YAML to a file (e.g. `clear-trust-bundle.yaml`) and applying this secret.
```